app/views/layouts/_footer.html.haml

```diff
@@ -20,8 +20,11 @@
           %h2
             = t("layout.footer."+key.to_s)
           %div
+
             - section_links.each do |section , link|
-              %a{:href => link, :target => "_blank"}
+              - anchor_key = "layout.footer.#{section}_anchor"
```

Comment:
it can be cleaner doing just this: ` - anchor = I18n.t("layout.footer.#{section}_anchor", default: "")`, but it needs to be tested.

Reply:
Indeed, it's cleaner that way. I tested it, works just fine